DependencyInjection/Neo4jExtension.php

```diff
@@ -167,20 +185,23 @@ private function getUrl(array $config): string
             $config['username'],
             $config['password'],
             $config['host'],
-            $config['port']
+            $this->getPort($config)
         );
     }
 
     /**
-     * {@inheritdoc}
+     * Return the correct default port if not manually set.
+     *
+     * @param array $config
+     *
+     * @return int
      */
-    public function getConfiguration(array $config, ContainerBuilder $container): Configuration
+    private function getPort(array $config)
```

Comment:
I did this in the `Configuration` in the first place but I prefer to limit the number of dependencies with external classes